interfaces/admin_cli.py

```diff
@@ -0,0 +1,453 @@
+            find_plan = plan_controller.find_plan(plan)
+            try:
+                plan_controller.view_plan_statistics(find_plan)
+                self.return_previous_page()
+                break
+            except ControllerError:
+                print(f"\033[31m * Plan {plan} not found. Please re-enter plan name. \033[00m")
+                continue
+
+    def do_close_plan(self, arg):
+        """
+        #3 Delete a plan
+        """
+        while True:
+            plan = input("Enter the plan name:")
+            find_plan = plan_controller.find_plan(plan)
+            try:
+                plan_controller.close_plan(find_plan)
+                self.return_previous_page()
+                break
+            except ControllerError:
+                print(f"\033[31m * Plan {plan} not found. Please re-enter plan name. \033[00m")
+                continue
+
+    def do_return_main_menu(self, arg):
+        AdminShell(self.user).cmdloop()
+
+
+class ManageVolunteerMenu(AdminShell):
+    """
+    ManageVolunteerMenu is the sub-class of command line interface for admin role.
+    It will be launched when the admin choose #3 and enter do_manage_volunteer_account().
+    """
+    admin_options = {"logout": 0, "create_volunteer": 1, "view_volunteer": 2,
+                     "edit_volunteer": 3, "deactivate_volunteer": 4, "reactivate_volunteer": 5,
+                     "delete_volunteer": 6,
+                     "return_main_menu": "r"}
+
+    def volunteer_menu(self) -> None:
+        print("\n\033[100m\033[4m\033[1m{}\033[0m\n".format("Manage Volunteer Accounts"))
+        print(
+            "[ 1 ] Create a new volunteer \n"
+            "[ 2 ] View a volunteer profile\n"
+            "[ 3 ] Edit a volunteer profile\n"
+            "[ 4 ] De-activate a volunteer Account\n"
+            "[ 5 ] Re-activate a volunteer Account\n"
+            "[ 6 ] Delete a volunteer Account\n\n"
+            "[ R ] Return to Main Menu\n"
+            "[ 0 ] Log-out\n")
+
+    def preloop(self) -> None:
+        """
+        Display manage volunteer menu.
+        :return:
+        """
+        self.volunteer_menu()
+
+    def return_previous_page(self) -> bool:
+        while True:
+            option = input("Enter R to return back to main menu. > ")
+            if option.isalpha() and option.upper() == 'R':
+                ManageVolunteerMenu(self.user).cmdloop()
+                break
+            else:
+                print("Invalid input. Please try again.")
+                continue
+
+    # ----- basic commands for volunteer account management -----
+    def do_create_volunteer(self, arg):
+        """
+        #1 Create a volunteer
+        """
+        print("\033[100m\033[4m\033[1m{}\033[0m ".format("Create a new volunteer account"))
+        username = input("Enter volunteer's username:")
+        password = input("Enter volunteer's password:")
+        firstname = input("Enter volunteer's firstname:")
```

Comment:
Could this be changed so that it says "first name" and in the next line, "last name"? Thanks!